encord/configs.py

```diff
@@ -190,3 +225,22 @@ def from_ssh_private_key(ssh_private_key: str, password: Optional[str], **kwargs
             return UserConfig(private_key, **kwargs)
         else:
             raise ValueError(f"Provided key [{ssh_private_key}] is not an Ed25519 private key")
+
+    @staticmethod
+    def from_env_variable(password: Optional[str] = "", **kwargs):
```

Comment:
Not sure if we need the extra method. Looking at the `from_ssh_private_key` member and then `from_env_variable` it seems like this does not use an ssh private key?!  

I think Rad gave a similar comment.